paper.tex

```diff
@@ -254,19 +254,21 @@ \subsubsection{Pointers to Data}
 
 \begin{ffcode}
 [ptr] > book
+  ptr > @
   ptr.block > title
     100
     [b] (b.as-string > @)
   ptr.block > price
     8
     [b] (b.as-int > @)
-heap.pointer > p1
-  0x1A76EC09 
+pointer. > p1
+  heap 1024
+  0
   108
 [p] > f
   seq > @
-    book (p.plus 7) > b
-    b.title.write "Object Thinking"
+    book (p.add 7) > b
+    b.title.write ("Object Thinking").as-bytes
```

Comment:
@Graur this line will be understood as 

```
(b.title.write ("Object Thinking")).as-bytes
```

better use two lines, for clarity